### ISSUE TITLE
Storm Distance and Bearing (still shows 0)

### ISSUE THREAD

**Issue by alpha520098** [Pirate-Weather/pirateweather#121]
Has anybody got this to work, I have priate weather for a short time and from the outset it's great so my gradtude to those that maintain it.

My question is that could someone tell me if they can get this to work?

**Comment by alexander0042** [comment]
Just a fun little update here, but the groundwork for this is officially working! It'll be tagged as "beta" for now, but the data is officially being processed and saved on my end

**Comment by alpha520098** [comment]
> Just a fun little update here, but the groundwork for this is officially working! It'll be tagged as "beta" for now, but the data is officially being processed and saved on my end

So when will this beta be released? 

And thank you for working on this - hopefully we can make some really good automations

**Comment by cloneofghosts** [comment]
> > Just a fun little update here, but the groundwork for this is officially working! It'll be tagged as "beta" for now, but the data is officially being processed and saved on my end
> 
> So when will this beta be released?
> 
> And thank you for working on this - hopefully we can make some really good automations

As of yesterday I noticed in this data in the v2 testing endpoint which I think you need to have dev endpoint access in order to view. You can try visiting `http://piratev2lb-a90c79daaddc2625.elb.us-east-1.amazonaws.com:8000/forecastv2/<APIKEY>/<LAT>,<LON>` to see if you have access to it. It's very much not production ready but I see this in the hourly section on the beta endpoint

```
nearestStormDistance": 159274.1875,
"nearestStormBearing": 218.65980529785156
```

As far as I'm aware V2 should launch sometime this winter and I'm not sure if there will be a open beta or something closer to launch.

**Comment by cloneofghosts** [comment]
@alpha520098 Yes, there is a beta running which you can try accessing using this URL: http://piratev2lb-a90c79daaddc2625.elb.us-east-1.amazonaws.com:8000/forecastv2/<APIKEY>/<LAT>,<LON>

It's still very much in progress and any feedback you have you can add here #155

**Comment by alpha520098** [comment]
> @alpha520098 Yes, there is a beta running which you can try accessing using this URL: http://piratev2lb-a90c79daaddc2625.elb.us-east-1.amazonaws.com:8000/forecastv2/APIKEY/LAT,LON
> 
> It's still very much in progress and any feedback you have you can add here #155

Thanks very much for the reply however this issue is that the URL that you have suppiled isnt working i keep getting this error:

Kong Error
Invalid authentication credentials.

any tips on how to make this work?

@cloneofghosts

**Comment by cloneofghosts** [comment]
@alpha520098 Like the regular API you need to supply your own APIKEY and LAT LON in the URL and it should work.

For example if your API key is 1234567890abcdefghijklmnopqrstuvwxyzABCD and you want to query data for 0,0 the URL would look like the following: `http://piratev2lb-a90c79daaddc2625.elb.us-east-1.amazonaws.com:8000/forecastv2/1234567890abcdefghijklmnopqrstuvwxyzABCD/0,0`

I tested with a free API key so it should work even if you're subscribed to the free tier.
